explanation/custom-images.md

```diff
@@ -0,0 +1,8 @@
+Building custom images of Anbox Cloud based on AAOS is possible but may get tricky.
+
+[note type="information" status="Note"]Building custom images is possible only for Android 13 and is currently only possible for AAOS images only.[/note]
+
+For example, using customised Anbox Cloud images will lead to more maintenance work, missing security updates provided by the Anbox Cloud team or requiring manual intervention to deploy Anbox Cloud. It is for these reasons that we generally don't recommend using a custom image unless you have a real business need for it.
+
+If you have a use case that demands an Anbox Cloud image with customisations applied, please write to us through the users forum.
```

Comment:
Should we keep it more generic? i.e.:
```suggestion
If you have a use case that demands an Anbox Cloud image with customisations applied, please contact us.
```

Possibly with a link to a contact form or something.

Reply:
I have used the same contact form that is on our product home page.